repo-policy/main.tf

```diff
@@ -49,7 +49,7 @@ module "tyk" {
     { branch    = "master",
       reviewers = "1",
       convos    = "false",
-    required_tests = ["Go 1.21.x Redis 5", "1.21-bullseye", "test (1.21.x, ubuntu-latest, amd64, 15.x)", "api-tests (sha256, mongo44)", "api-tests (sha256, postgres15)", "api-tests (murmur64, mongo44)", "api-tests (murmur64, postgres15)"] },
```

Comment:
I feel that 1.21-bullseye can also be removed as api-tests depend on this job to pass.

Reply:
No @alephnull , 1.21-bullseye is goreleaser job, I don't think we want to remove that

Reply:
api-tests depends on goreleaser so it seems pointless to keep 1.21-bullseye.